vertx-grpc-docs/src/main/asciidoc/plugin.adoc

```diff
@@ -117,6 +117,7 @@ The plugin accepts the following arguments:
 - _--grpc-service_: generate the service files
 - _--grpc-io_: generate stub files compatible with `io.grpc:grpc-stub`, not generated by default
 - _--grpc-transcoding_: whether to generate transcoding options for methods with HTTP annotations
+- _--service-prefix_: generate service classes with a prefix
```

Comment:
I think we need to explain that the original gRPC service name is prefixed to avoid confusion and that it will impact the generation of all classes, e.g. if prefix is Foo, then we would have FooGreeter, FooGreeterGrpcClient etc... and give an example in the doc so users understand easily